maybe/maybe_unix.go

```diff
@@ -0,0 +1,19 @@
+package maybe
+
+import (
+	"io/ioutil"
+	"os"
+	"runtime"
+
+	"github.com/google/renameio"
+)
+
+// WriteFile mirrors ioutil.WriteFile. On Linux it replaces an existing file
+// with the same name atomically. On Windows the file is not written atomically
+// and most permission bits are ignored.
```

Comment:
Could you rephrase this to state “on Windows files cannot be written atomically”?

The current phrasing doesn’t convey that this is a technical limitation.

Ideally, also add a reference to the discussion where this was explained.